core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2665,8 +2665,27 @@ object SparkContext extends Logging {
     // When running locally, don't try to re-execute tasks on failure.
     val MAX_LOCAL_TASK_FAILURES = 1
 
+    // SPARK-26340: Ensure that executor's core num meets at least one task requirement.
+    def checkCpusPerTask(
+      clusterMode: Boolean,
+      maxCoresPerExecutor: Option[Int]): Unit = {
+      val cpusPerTask = sc.conf.get(CPUS_PER_TASK)
```

Comment:
Missing `contains` check, here ?

Reply:
@Ngone51 `CPUS_PER_TASK` has default value 1, so, contains check is not needed.